_docs/services/relational-database.md

```diff
@@ -109,6 +124,24 @@ If the response is `OK` instead of `FAILED` then your database is ready to use.
 
 The cloud.gov team aims to provide clearer status indicators in a future release of our service broker.
 
+## Update an instance
+
+To update an existing service instance run the following command:
+
+```sh
+cf update-service aws-rds ${SERVICE_NAME} -p ${SERVICE_PLAN_NAME}
```

Comment:
Wondering of the placeholder shouldn't be `NEW_SERVICE_PLAN_NAME` or `UPDATED_SERVICE_PLAN_NAME` to avoid confusion.

Reply:
Good call!